### ISSUE TITLE
Keras Model Import Problem:  NullPointerException 

### ISSUE THREAD

**Issue by rogerlee24** [deeplearning4j/deeplearning4j#9543]
#### Issue Description

When I try to import my pre-trained keras tensorflow model, I encounter a NullPointerException.

` String path = new ClassPathResource("my_model.h5").getFile().getPath();

  MultiLayerNetwork modelAndWeights = KerasModelImport.importKerasSequentialModelAndWeights(path);`

I am not sure if it is related  #9524  and #8827

#### Version Information

Keras Version: 2.7.0
Tensorflow Version: 1.15.0
Keras backend: tensorflow

Relevant versions
dl4j-examples version: 1.0.0-SNAPSHOT
Platform information (OS, etc): Windows 10

#### Additional Information

Stack trace
`Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.CommandLineWrapper.main(CommandLineWrapper.java:66)
Caused by: java.lang.NullPointerException
	at org.deeplearning4j.nn.modelimport.keras.layers.normalization.KerasBatchNormalization.<init>(KerasBatchNormalization.java:124)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:257)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:225)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:110)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:57)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildSequential(KerasModelBuilder.java:326)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:218)
	at org.deeplearning4j.examples.KerasImport.main(KerasImport.java:15)
	... 5 more`

When converting BatchNormalization, the error occurred, it's details.

![image](https://user-images.githubusercontent.com/57137841/143879844-c27b8948-62fb-4771-8b93-b15fb0faad50.png)
![image](https://user-images.githubusercontent.com/57137841/143879899-393d0a32-bb6e-4258-945e-c298c620863d.png)

I am so sorry for my bad Englsih, can you help me solve the problem?
Thanks!

**Comment by agibsonccc** [comment]
@rogerlee24 it looks like this is a separate namespace for model weights. Could you confirm how this model was buitl?

**Comment by agibsonccc** [comment]
@rogerlee24 it looks like this is a fairly recent change to the file format: https://github.com/keras-team/keras/blob/master/keras/saving/hdf5_format.py#L668 I guess now the weights are saved in  their own group than with the layers. I'll have to look further in to this.

**Comment by rogerlee24** [comment]
@agibsonccc 
it's a simple model.
![image](https://user-images.githubusercontent.com/57137841/143970694-86c176ed-0f0c-42b8-ab2d-dfaf301dfd19.png)
![image](https://user-images.githubusercontent.com/57137841/143970644-7eb235b4-aeb5-48ac-8f87-096f7d6f73e4.png)

**Comment by agibsonccc** [comment]
@rogerlee24 "it's simple" is not really what I'd be looking for. After a bit of digging it turns out that there is a change in the model format saved by keras that we'll need to handle. Luckily another user gave me a model and I was able to find the problem. I'll have a fix up in the next day or 2. No matter how simple a model is if keras changes its way of saving models a bug will occur :) try downgrading and seeing if that helps. I think this is an issue with keras 2.7.0 or above.

**Comment by rogerlee24** [comment]
@agibsonccc  sry, I misunderstood what you mean. Thanks for your  guidance!

**Comment by agibsonccc** [comment]
@rogerlee24 sure no problem. Generally when I mean model
, , I usually would want the model file so I can attempt to load it. When working with source code, I have to set it up in my ide, double check the configuration and may not even be running the right version.

It just creates a lot of work when trying to understand a problem. Thanks for your patience as I get this resolved.

**Comment by agibsonccc** [comment]
Closing in favor of https://github.com/eclipse/deeplearning4j/issues/9524 - The OP did confirm this was the same issue.

**Comment by tuhuu** [comment]
I have faced the same problem and I added it to build the model. This solved my issue.
`model = models.Sequential()`
